cmd/rollback.go

```diff
@@ -61,6 +61,11 @@ func rollback(cmd *cobra.Command, args []string) error {
 	}
 
 	response, err := aBsys.Rollback(checkOnly)
+	if err != nil {
+		cmdr.Error.Println(err)
+		os.Exit(2)
+		return err
```

Comment:
I'm not too familiar with Go, but wouldn't this mean that L86 will never get to print out the error code since we're directly printing it out here and returning it?

Reply:
That is true. The ROLLBACK_FAILED state is pointless anyway, since the state of response must not be checked if err != nil.

Reply:
I will remove it when I get the time.